META.md

```diff
@@ -96,3 +108,135 @@ meta exec "git commit -s -m 'Removed integtest.sh.'"
 meta exec "git push username remove-integtest-sh"
 meta exec "gh pr create --title 'Removing default integtest.sh.' --body='Coming from https://github.com/opensearch-project/opensearch-build/issues/497, removing default integtest.sh.'"
 ```
+
+### Increment a Version in Every Plugin
+
+Because one cannot install an older patch version of a plugin on a newer OpenSearch-min (see [OpenSearch#1707](https://github.com/opensearch-project/OpenSearch/issues/1707)), it's common to have to increment versions in all plugins, without making other changes. This was the case in [the 1.2.3 release](https://github.com/opensearch-project/opensearch-build/issues/1365).
```

Comment:
The wording should be genetic to any old version of plugins and not specific to patch version